lib/bdev/bdev.c

```diff
@@ -412,6 +412,8 @@ __submit_request(struct spdk_bdev *bdev, struct spdk_bdev_io *bdev_io)
 	if (bdev_io->type == SPDK_BDEV_IO_TYPE_RESET) {
 		spdk_bdev_cleanup_pending_buf_io(bdev);
 		ch = NULL;
+	} else if (bdev_io->type == SPDK_BDEV_IO_TYPE_PASSTHRU) {
+		ch = NULL;
```

Comment:
For SPDK_BDEV_IO_TYPE_NVME_IO, we would use the channel, but it is not required for SPDK_BDEV_IO_TYPE_NVME_ADMIN.